typeset/tests/typeset.test.js

```diff
@@ -60,20 +61,47 @@ afterAll(() => {
   }
 })
 
-test('Fail if user provide wrong path for input file.', async (done) => {
-  const res = await converter.createMapOfMathMLElements(log, './wrong/path.xhtml', pathToCss, pathToOutput, 'html', 3000)
+test('Fail if user provides wrong path for input file (Highlight).', async (done) => {
+  const res = await converter.highlightCodeElements(log, './wrong/path.xhtml')
+  expect(res).toBe(converter.STATUS_CODE.ERROR)
+  done()
+})
+
+test('Check if `pre` elements with lang attribute are highlighted', async (done) => {
```

Comment:
I believe that this test case covers invalid lang attributes on `pre` tags because if a pre tag has a lang attribute but no elements were highlighted, that means the lang attribute was invalid. 

Is my thinking correct here or am I off? @philschatz